src/components/WindowViewSettings.js

```diff
@@ -63,10 +67,11 @@ export class WindowViewSettings extends Component {
         none of the click handlers work? */
     const menuItem = ({ value, Icon }) => (
       <ViewOption
-        selected={windowViewType === value}
-        key={value}
+        aria-selected={windowViewType === value}
```

Comment:
Is aria-selected a valid/useful attribute for menu items? I'm not seeing it on https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/menuitem_role. Maybe this should be a `menuitemradio` + `aria-checked`?

Reply:
Ah thanks for spotting :) 
i added `aria-checked` and `menuitemrole`